packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -5,18 +5,21 @@ import {
     CSSRules,
 } from "@microsoft/fast-jss-manager";
 import { applyLocalizedProperty, Direction } from "@microsoft/fast-jss-utilities";
-import { hoverContrast } from "../utilities/colors";
+import {
+    neutralFillStealthHover,
+    neutralFillStealthRest,
+    neutralForegroundRest,
+    neutralOutlineRest,
+} from "../utilities/color";
 import { CarouselClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
 
 const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<CarouselClassNameContract, DesignSystem> => {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
     const direction: Direction = designSystem.direction;
-    const white: string = "white";
-    const darkGray: string = "#101010";
-    const lightGray: string = "rgb(206, 206, 206)";
-    const lightGrayRest: string = "rgba(206, 206, 206, 0.6";
+    const white: string = "#FFF";
+    const black: string = "#111";
```

Comment:
Why `#111`? Is this tied to foreground?

Reply:
Seems like we'd want to avoid hard-coding this, or maybe this is what Chris mentioned the other day about needing black and white constants globally. If we must, please set it to #101010 for now.

Reply:
@nicholasrice didn't we add constants for black and white in the v4 color work?

Reply:
Filed an issue here: https://github.com/Microsoft/fast-dna/issues/1502

Reply:
@bheston Would these values align to the L{1-4} swatches you have been exploring? What is the logical source of these colors?